coreutils/plan.sh

```diff
@@ -20,14 +20,11 @@ pkg_deps=(
   core/libcap
 )
 pkg_build_deps=(
-  core/coreutils
-  core/diffutils
```

Comment:
Maybe the same as m4,  but without diffutils the number of tests run is significantly smaller (order of magnitude).  I'm not sure if the underlying builds are different yet or if it's strictly related to do_check.

Reply:
Looks like diffutils is required in this case 
```
checking whether perror matches strerror... ./configure: line 61057: cmp: command not found
no
```

I'll take a deeper look to see if there's a build flag we can pass to bypass that check. 

I also noted the same output for `hostname`. I'm hopeful that's something we can set as I suspect that it's only used for logging/marking builds somehow.